index.js

```diff
@@ -62,22 +62,31 @@
          *
          * @param HTMLElement element Element to inject script into.
          * @param String file         URL, relative or absolute, of JavaScript file.
-         * @param Function callback   Callback when script has loaded.
+         * @param Function [success]  Function called after the script has loaded successfully.
+         * @param Function [error]    Function called if an error occurred while loading the
+         *                            script.
          * @return HTMLScriptElement
          */
-        injectJavaScript: function (element, file, callback) {
+        injectJavaScript: function (element, file, success, error) {
             var
                 script = element.ownerDocument.createElement('script'),
                 src = ensureAbsolute(file);
 
-
-            if (typeof callback == 'function') {
+            if (typeof success === 'function') {
                 script.onload = function (e) {
-                    callback(file);
+                    success(file);
                     script.onload = null;
                 };
             }
 
+            if (typeof error === 'function') {
+                script.onerror = function (e) {
+                    error(file);
```

Comment:
Or forget that. It seems these element specific `onerror` callbacks only receive a plain event object with no real information about what went wrong.

But you can remove the argument since we don't use it (and if you'd clean up the `onload` callback too that would be neat).